packages/lib-classifier/src/store/annotations/DrawingAnnotation.js

```diff
@@ -0,0 +1,15 @@
+import { types } from 'mobx-state-tree'
+import Annotation from './Annotation'
+import { PointAnnotation } from './DrawingToolAnnotations'
+
+const Drawing = types.model('Drawing', {
+  value: types.array(types.union({
+    dispatcher: (snapshot) => {
+      if (snapshot.toolType === 'point') return PointAnnotation
```

Comment:
This is fine for now but is going to get clunky when we want to add new tools.

Reply:
Yes plan to refactor to switch statement at minimum when add next tool.

Reply:
Longer term, this should be generated dynamically from the list of available tools, otherwise developers have to remember to touch the code in multiple places when adding a new drawing tool.